### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Aryanutkarsh**
<h2>Improved the Readability </h2>

- Fixed the broken link to the research paper
- Center aligned the tables(to improve readability)
- Fixed grammatical error

### CODE REVIEW

**Comment by Xrenya** [comment]
@Aryanutkarsh it should be `DINOv2 models produce` without `s`. It was correct.

**Comment by patricklabatut** [comment]
> Fixed the broken link to the research paper

Note there is already a paper link in the link bar just above, so I would perhaps not call this broken and I am not sure this is exactly needed (but fine).

> Center aligned the tables

That's also not exactly critical (but again fine). However this does not seem to work as intended, some of the contents of the table (e.g. "ViT-g/14") gets centered as well (using GitHub's markdown preview).

File: README.md [hunk 1]
```diff
@@ -11,10 +11,13 @@ Vasil Khalidov,
 Patrick Labatut,
 Armand Joulin,
 Piotr Bojanowski
-
+<div align="center">
+  
 [[`Paper`](https://arxiv.org/abs/2304.07193)] [[`Blog`](https://ai.facebook.com/blog/dino-v2-computer-vision-self-supervised-learning/)] [[`Demo`](https://dinov2.metademolab.com)] [[`BibTeX`](#citing-dinov2)]
+  
+</div>
```
Comment (patricklabatut) [inline README.md:18]: Let's not center this link bar.

**Review by patricklabatut — CHANGES_REQUESTED** [review]
Thanks, back to you to address the comments.

File: README.md [hunk 2]
```diff
@@ -26,7 +27,7 @@ https://user-images.githubusercontent.com/60359573/230078733-5faffa19-e6ce-4c55-
 </div>
 
 ## Pretrained models
-
+<div align="center">
 <table>
```
Comment (patricklabatut) [inline README.md:31]: After doing a bit of research, it seems styling the table to auto-center is the proper way to achieve this:

```
<table style="margin: auto">
```

File: README.md [hunk 3]
```diff
@@ -26,7 +27,7 @@ https://user-images.githubusercontent.com/60359573/230078733-5faffa19-e6ce-4c55-
 </div>
 
 ## Pretrained models
-
+<div align="center">
```
Comment (patricklabatut) [inline README.md:30]: Nit: keep an empty line between title and table thanks.

File: README.md [hunk 4]
```diff
@@ -187,7 +189,7 @@ python dinov2/run/eval/linear.py \
 ```
 
 We release the weights from evaluating the different models:
-
+<div align="center">
```
Comment (patricklabatut) [inline README.md:192]: Same comment as above on styling the table instead (and please keep a separating blank line).

File: README.md [hunk 5]
```diff
@@ -11,10 +11,11 @@ Vasil Khalidov,
 Patrick Labatut,
 Armand Joulin,
 Piotr Bojanowski
-
+  
 [[`Paper`](https://arxiv.org/abs/2304.07193)] [[`Blog`](https://ai.facebook.com/blog/dino-v2-computer-vision-self-supervised-learning/)] [[`Demo`](https://dinov2.metademolab.com)] [[`BibTeX`](#citing-dinov2)]
+
```
Comment (patricklabatut) [inline README.md:16]: Nit: Remove left-over whitespace from the previous changes.

**Comment by Aryanutkarsh** [comment]
@patricklabatut sir thank you so much for being so understanding and guiding. I am new to open source,I want work with people like you rather making pr to first contribution repos.This thought me how to resolve the reviews 

### Thank you Sir 🙂

**Comment by patricklabatut** [comment]
@Aryanutkarsh Remove the div elements that were introduced, I don't think they should not be needed anymore with the "margin: auto" styling. Test and if that's indeed fine, that should be good to merge.

**Comment by Aryanutkarsh** [comment]
Done sir 👍

**Review by patricklabatut — APPROVED** [review]

**Pull request merged**